### DISCUSSION TITLE
Formatting action starts failing due to wrong newlines

### DISCUSSION THREAD

**Discussion by BearToCode** [community/community#79380] [Actions] [answered]
### Select Topic Area

Bug

### Body

I have a workflow that use [prettier](https://prettier.io/) to ensure that code in my repo has the same style. 
This action has recently started failing, with the same files that worked around a week ago. There have been no changes to the build process/other files, I just re-run the same workflow. If I try to run the same check locally, no errors show up. 
I'm pretty sure this is due to some different newline format being used.

Has GitHub Actions received any update that may explain this behaviour? I'm struggling to figure out a way to fix this, as I do not know the root of the problem. It is like the Action changes(or no longer does) newlines when checking out the repo.

[Here](https://github.com/BearToCode/carta/actions/runs/7150583541/job/19710492759) is the workflow that is causing the problem.
Thanks for any help.

**Comment by BearToCode** [comment, marked as answer]
Solved! This was to due prettier being updated to 3.1.1, which somehow caused this. Pinning prettier to v3.1.0 fixed it.

**Reply by queenofcorgis** [reply]
@BearToCode thanks for coming back to let us know the solution. We’re glad you could get this solved and we want to make sure others can do the same so we’ve reopened this discussion for better discoverability for our members.

**Comment by saCadOww** [comment]
- uses: actions/checkout@v4
  - name: Setup PowerShell module cache
    id: cacher
    uses: actions/cache@v3
    with:
      path: "~/.local/share/powershell/Modules"
      key: ${{ runner.os }}-SqlServer-PSScriptAnalyzer
  - name: Install required PowerShell modules
    if: steps.cacher.outputs.cache-hit != 'true'
    shell: pwsh
    run: |
      Set-PSRepository PSGallery -InstallationPolicy Trusted
      Install-Module SqlServer, PSScriptAnalyzer -ErrorAction Stop
[Testing your code](https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-powershell#testing-your-code)

**Comment by saCadOww** [comment]
- uses: actions/checkout@v4
  - name: Setup PowerShell module cache
    id: cacher
    uses: actions/cache@v3
    with:
      path: "~/.local/share/powershell/Modules"
      key: ${{ runner.os }}-SqlServer-PSScriptAnalyzer
  - name: Install required PowerShell modules
    if: steps.cacher.outputs.cache-hit != 'true'
    shell: pwsh
    run: |
      Set-PSRepository PSGallery -InstallationPolicy Trusted
      Install-Module SqlServer, PSScriptAnalyzer -ErrorAction Stop
[Testing your code](https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-powershell#testing-your-code)

**Comment by saCadOww** [comment]
- uses: actions/checkout@v4
  - name: Setup PowerShell module cache
    id: cacher
    uses: actions/cache@v3
    with:
      path: "~/.local/share/powershell/Modules"
      key: ${{ runner.os }}-SqlServer-PSScriptAnalyzer
  - name: Install required PowerShell modules
    if: steps.cacher.outputs.cache-hit != 'true'
    shell: pwsh
    run: |
      Set-PSRepository PSGallery -InstallationPolicy Trusted
      Install-Module SqlServer, PSScriptAnalyzer -ErrorAction Stop
[Testing your code](https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-powershell#testing-your-code)### 

> [`_****_`](url)

**Comment by saCadOww** [comment]
Sacad### ### 

> _****_
